mimic/test/dummy.py

```diff
@@ -96,63 +101,61 @@ def resource(self):
         return example_resource
 
 
-@implementer(IEndpointTemplate)
-class ExampleEndpointTemplate(object):
+def ExampleEndpointTemplate(name=u"example", region="EXTERNAL", version="v1",
```

Comment:
Coding-standard; since this is a function and not a type the first letter should be lower-case.  Thanks for exposing it as a function though, it's nice to not have to support inheritance as an API surface.